scripts/india_wris/India_WRIS_Rainfall/README.md

```diff
@@ -0,0 +1,23 @@
+## Rainfall Dataset - India WRIS
+
+Water Resources Information System (WRIS): Rainfall dataset provides information on actual, normal and deviation of rainfall data at the district level.
+
+The rainfall parameters represented in this dataset are:
+
+- Normal rainfall (Average rainfall for the given location)
+- Actual rainfall (Actual rainfall measured in that location)
+- Deviation (Percentage change between actual and normal rainfall measurements)
+
+Node properties can be found in the [mcf](./India_WRIS_Rainfall.mcf) and [tmcf](./India_WRIS_Rainfall.tmcf) files.
+
+### Dataset info
+
+- Spatial resolution - District Level
+- Observation Period - Monthly
+- Data availability - 2011 to 2021
+
+### Raw data download
+
+Source link: [https://indiawris.gov.in/wris/#/DataDownload](https://indiawris.gov.in/wris/#/DataDownload)
+
+
```

Comment:
In India_WRIS_Rainfall, we don't have any scripts. Since the data is small and there are only three statvars, I did the preprocessing manually. But I added commands to run the code in other two directories: [India_WRIS_Surface](https://github.com/datacommonsorg/data/blob/9e1ab51d6e1551ff631a7f65492e92af9f7d8c26/scripts/india_wris/India_WRIS_Surface/README.md) and [India_WRIS_Ground](https://github.com/datacommonsorg/data/blob/9e1ab51d6e1551ff631a7f65492e92af9f7d8c26/scripts/india_wris/India_WRIS_Ground/README.md).

Reply:
could you please add the detailed steps you did to the required data? e.g. int the link to download data, there are some options (which application / report), then what were the manual steps you did to that data? imagine someone else needs to come a year from now to refresh the data, what would they need to know?